chirpstack-concentratord-sx1302/src/config/mod.rs

```diff
@@ -280,6 +285,7 @@ fn example_configuration() -> Configuration {
         gateway: Gateway {
             lorawan_public: true,
             model: "rak_2287_eu868".to_string(),
+            gateway_id: "0000000000000000".to_string(),
```

Comment:
Should be `"".into(),`

Reply:
Changed to `None` because `gateway_id` is now an `Option`.